src/components/Portal/Portal.tsx

```diff
@@ -103,7 +104,7 @@ class Portal extends AutoControlledComponent<ReactPropsStrict<PortalProps>, Port
     onUnmount: PropTypes.func,
     open: PropTypes.bool,
     trigger: PropTypes.node,
-    triggerRef: PropTypes.func,
+    triggerRef: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
```

Comment:
probably, this is a good candidate for custom prop types - would expect this to appear in multiple places